### PR TITLE
Remove contracts folder and update README

### PR DESCRIPTION
**Pull request by pfletcherhill**

### CODE REVIEW

File: app/components/App.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import Header from "./Header";
 import Home from "./Home";
 import Auction from "./Auction";
 import About from "./About";
+import AuctionList from "./AuctionList";
```
Comment (mertcelebi) [inline app/components/App.js:9]: Did you add this for development reasons?

**Review by mertcelebi — COMMENTED** [review]
Quick question

File: app/components/App.js (hunk 1 above)
Comment (pfletcherhill) [reply app/components/App.js:9]: No, the home page will just have information about the current auction (like a storefront), and then `AuctionList` will have all auctions (at `/auctions`).

Comment (mykelp) [reply app/components/App.js:9]: Just remove this. No need for this rn

**Pull request merged**